modules/lvms-limitations-for-creating-snapshots-and-clones.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-limitations-for-creating-snapshots-and-clones_{context}"]
+= Limitations for creating volume snapshots and volume clones
+
+{lvms} has the following limitations for creating volume snapshots and volume clones in multi-node topology:
+
+* Creating volume snapshots and volume clones is based on the LVM thin pool capabilities.
+* The node must have additional storage after creating a volume snapshot and volume clone for further updating the original data source.
+* Volume snapshots and volume clones can be created only on the node where you have deployed the original data source.
```

Comment:
```suggestion
* Volume snapshots and volume clones get created only on the node where you have deployed the original data source.
```

To avoid passive voice. Please check with SME for technical accuracy though.

Reply:
Meaning is unaffected by this, change is ok from my side

Reply:
@jakobmoellerdev 
As volume clones and snapshots can be created by users, can we write:

You can create volume snapshots and volume clones only on the node where you have deployed the original data source.

Reply:
Thank you @jakobmoellerdev, @Srivaralakshmi for the review.
Updated the line.

Reply:
@Srivaralakshmi -  If we can proceed to merging this PR, could you please add the "peer-review-done" label?
Thanks.